src/UI/Dependency/Dependency.tsx

```diff
@@ -52,6 +57,7 @@ export const DependencyContext = createContext<Dependencies>({
   authHelper: new DummyAuthHelper(),
   archiveHelper: new DummyArchiveHelper(),
   authController: new DummyAuthController(),
+  useAuth: defaultAuthContext,
```

Comment:
I take you'll rename it to `authHelper`, or `authManager `to stay in line with the rest after? useAuth hints at a hook, which can be confusing

Reply:
Yes, I planned to use the authController, after I remove old one and free the name, but I can change it to any of those right now